src/engine/Settings.cs

```diff
@@ -63,10 +63,11 @@ public enum StrainBarVisibility
     // Graphics Properties
 
     /// <summary>
-    ///   Sets whether the game window is in fullscreen mode
+    ///   Sets window mode of the game window
     /// </summary>
     [JsonProperty]
-    public SettingValue<bool> FullScreen { get; private set; } = new(true);
+    public SettingValue<DisplayServer.WindowMode> DisplayMode { get; private set; } =
```

Comment:
I think we should have our own enum to guard against Godot changing theirs and messing stuff up.
I think we'll need just Windowed, Fullscreen and ExclusiveFullscreen values.